app/src/main/kotlin/com/bitwarden/authenticator/data/authenticator/manager/TotpCodeManager.kt

```diff
@@ -14,16 +16,9 @@ interface TotpCodeManager {
     /**
      * Flow for getting a DataState with multiple verification code items.
      */
-    fun getTotpCodesStateFlow(
-        itemList: List<AuthenticatorItemEntity>,
-    ): StateFlow<DataState<List<VerificationCodeItem>>>
-
-    /**
-     * Flow for getting a DataState with a single verification code item.
-     */
-    fun getTotpCodeStateFlow(
-        item: AuthenticatorItemEntity,
-    ): StateFlow<DataState<VerificationCodeItem?>>
+    fun getTotpCodesFlow(
+        itemList: List<AuthenticatorItem>,
+    ): Flow<List<VerificationCodeItem>>
```

Comment:
Introducing our two new domain models!

`AuthenticatorItem`: (everything required to generate an actual verification code, along with `Source` data which is `Shared` or `Local`.

and `VerificationCodeItem`: (model that gets sent to the VM and has actual code info so its ready for display)